db/migrate/20241011071014_update_item_name_to_product_in_od_report.rb

```diff
@@ -0,0 +1,39 @@
+class UpdateItemNameToProductInODReport < ActiveRecord::Migration[7.0]
+  # OD: Orders and Distributors
+  def up
+    # adding subtype filter just to be safe
+    options = ReportRenderingOptions.where(report_type: 'orders_and_distributors', report_subtype: nil)
+
+    options.find_each do |option|
+      begin
+        fields_to_show = option.options[:fields_to_show]
+        next if fields_to_show&.exclude?('item_name')
+
+        fields_to_show.delete('item_name')
+        fields_to_show << 'product'
+        option.update(options: option.options)
```

Comment:
I guess this works if `fields_to_show` is a pointer to the value inside the options hash. 
If that's the case, I think you should be able to simply call:
```suggestion
        option.save
```

But either way, I'm not sure as it's an uncommon way to do things, so I would like to see a test to check that it works.